hoot-services/src/main/java/hoot/services/controllers/osm/user/UserResource.java

```diff
@@ -347,6 +347,7 @@ public Response getPrivileges(@Context HttpServletRequest request) {
      * @return true if user has admin privileges, else false
      */
     public static boolean adminUserCheck(Users user) {
+        if (user == null) return false;
         Map<String, String> privileges = PostgresUtils.postgresObjToHStore(user.getPrivileges());
         return privileges != null && ("true").equals(privileges.get("admin"));
```

Comment:
since the call of postgresObjToHStore() always returns a map object, the null check isn't necessary since it will always be true